### ISSUE TITLE
Resource displayName is ignored when generating python stub

### ISSUE THREAD

**Issue by FastGeert** [Jumpscale/go-raml#448]
Inspecting the files after generating python stubs, I notice that filename of the python stub for my resource is chosen by the path of the resource on the server:

```raml
/api/v1/root/{path:*}:
...
```

This resulted in the following python code:
```python
class Client:
    def __init__(self, base_uri=""):
        http_client = HTTPClient(base_uri)
        self.apiv1rootpath = Apiv1rootpathService(http_client)
        self.close = http_client.close
```

Not very nice.

I noticed that I did not add a displayName to customize a nice name for this resource, so I did, but that did not alter the generated source code.

**Comment by iwanbk** [comment]
Yes, the display name only work on method level.

**Comment by iwanbk** [comment]
Fixed at #450
